intraday_analyzer/analyzer.py

```
@@ -0,0 +1,677 @@
+            signals.append(0)
+
+        # Price vs VWAP
+        if inp.current_price > inp.price_summary.vwap:
+            signals.append(1)
+        elif inp.current_price < inp.price_summary.vwap:
+            signals.append(-1)
+        else:
+            signals.append(0)
+
+        score = sum(signals)
+        total = len(signals)
```

Ruff (0.14.14)

[error] Local variable `total` is assigned to but never used

Remove assignment to unused variable `total`

(F841)